sdk/core/src/index.ts

```diff
@@ -203,6 +204,8 @@ var ReactMoE = {
   /**
    * Sets the unique id of the user. Should be set on user login.
    * @param uniqueId unique id to be set
+   * 
+   * @deprecated This function is deprecated in favour of identifyUser(). This function will be removed in 13.0.0"
    */
   setUserUniqueID: function (uniqueId: string) {
```

Comment:
Let's add a log for deprecated method usage

Reply:
added